### PR TITLE
created Contributing.md 

### PR DESCRIPTION
**Pull request by Pallavi024**
NOTE:
- Please fill out the template below 

  - [ ] Did you read the instructions in the [WORKFLOW](https://github.com/wtfpython-web/wtfpython-web/wiki/Workflow) ?

  - [ ] Is your Python code styled according to the PEP8 code style convention as mentioned [here](https://www.python.org/dev/peps/pep-0008/) ?

  - [ ] Have you run the tests ?

  - [ ] Have you followed commit guidelines as mentioned [here](https://github.com/wtfpython-web/wtfpython-web/wiki/Commit-Guidelines) ?

### CODE REVIEW

**Comment by realslimshanky** [comment]
@Pallavi024 please squash your commits.

**Comment by Pallavi024** [comment]
@realslimshanky I have squashed my commits...please take a look at it :smile:

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+
```
Comment (realslimshanky) [inline Contributing.md]: Please add a heading here like 'Contribution Guide - WTF'

Comment (Pallavi024) [reply Contributing.md]: @realslimshanky  I apologise for the delay.I have made the required changes

File: Contributing.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+# Step 4. Sending Your Changes
+
+### Caution
+
+Before committing your changes, please check that you are indeed in a development branch created in step 3. To check if you are in a branch, type:
+
+$ git branch
+
+Your current branch will have an asterisk (*) next to it. Ensure that there is no asterisk next to the master branch.
```
Comment (satwikkansal) [inline Contributing.md]: This asterisk should be escaped. It is making all the trailing text in Italics style.

File: Contributing.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
+# Contribution Guide -WTF
```
Comment (satwikkansal) [inline Contributing.md]: Missing space around `-`

Comment (satwikkansal) [reply Contributing.md]: Also `WTFpython contribution guide` sounds better to me.

**Review by realslimshanky — APPROVED** [review]
LGTM

**Pull request merged**